siliconcompiler/flows/fpgaflow.py

```diff
@@ -86,7 +86,10 @@ def setup(chip, flowname='fpgaflow'):
 
     flowtools = setup_frontend(chip)
     for step in flowpipe:
-        flowtools.append((step, tool_lookup(flow, step), step))
+        task = step
+        if (step == 'syn_vpr') and (flow == 'vpr'):
```

Comment:
can't we just change the pipe to just have 'syn' (migth want to call is `syn_fpga` so the `syn_asic` can be differentiated

Reply:
The single fpgaflow will need to change significantly if we are to take advantage of modules.